.gitignore

```diff
@@ -1,2 +1,3 @@
-target/
-dependency-reduced-pom.xml
+target/
+dependency-reduced-pom.xml
+/config/nbproject/
```

Comment:
nit: could you add this on your global git ignore? https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
ok

Reply:
@timurgen by adding `/config/nbproject/` to your global gitignore, then it could be removed from here :)